keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -208,39 +209,46 @@ type RequeueState struct {
 }
 ```
 
-Add a new field `evictionStats` to the WorkloadStatus struct to track the number of evictions by type and reason:
+Add a new field, `schedulingState`, to the WorkloadStatus struct to track the number of scheduling statistics.
+SchedulingState serves as a history of the Workload thus will not be reset.
+First sub-resource to be a part of the `schedulingState` will be `evictionStates` a struct to track the number of evictions by reason.
```

Comment:
```suggestion
First sub field to be a part of the `schedulingState` will be `evictionStates` a struct to track the number of evictions by reason.
```

The `sub-resource` has a dedicated technical term which indicates `status` and `scale`, something else.